<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserShowHistory</class>
 <widget class="QWidget" name="UserShowHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>140</y>
     <width>991</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查看详情</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>按日期升序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按日期降序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按金额升序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按金额降序</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>按时间区间搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>574</x>
     <y>50</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>UserShowHistory</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
